frontend.java

```diff
@@ -0,0 +1,540 @@
+		    			totalAmount += amount;
+		    			if((totalAmount) > 1000){
+		    				return false;
+		    			}
+		    			userAccount = false;
+		    		}
+		    		itemNum++;
+		    	}
+		    }
+		}
+		return true;
+	}
+	public static void front_withdraw() {
+		//get account number
+		//get amount
+		System.out.print("Account Number:");
+		String accountNumber = frontend.input();
+		if(accountNumber(accountNumber)==false){
+			frontend.error("Error: Invalid account number.");
+		}else{
+			if(accountExists(accountNumber)==false){
+				frontend.error("Error: Invalid account number.");
+			}else{
+				System.out.print("Account Amount:");
+				String accountAmount = frontend.input();
+				if(accountAmount(accountAmount)==false){
+					frontend.error("Error: Invalid account amount.");
+				}else{
+
+					if(lowerLimit(accountAmount)==false){
+						frontend.error("Error: Invalid account amount. Must be greater than 0.");
+					}else{
+						if(upperLimit(accountAmount)==false){
+							frontend.error("Error: Invalid account amount. The amount is too high for the type of login.");
+						}else{
+							if(withdrawLimit(accountNumber, accountAmount)==false){
+								frontend.error("Error: Invalid account amount. The amount is too high for the type of login.");
+							}else{
+								//convert amountStr to int well more validate
+								//make sure the account number is long enough
+								//Does there have to be an account name?
+								//make sure they are accounts
+								
+								//Pad Zeros on account number, account amount
+								accountAmount = addZeros(accountAmount, 8);//adds zeros in front if size is less than 8
+								accountNumber = addZeros(accountNumber, 6);//adds zeros in front if size is less than 6
+
+										
+								String str = "02_" + accountNumber + "_" + accountAmount; 
+								frontend.transactionFile.add(str);
+								//write.writeToSummaryFile(str);
+								System.out.println("Successful Withdraw.");
+							}
+						}
+					}
+				}
+			}
+		}
+	}
+	public static void writeToAccountsFile(String content){
+		try {
+			 
+			File file = new File("./src/valid_accounts.txt");
+ 
+			// if file doesnt exists, then create it
+			if (!file.exists()) {
+				file.createNewFile();
+			}
+ 
+			FileWriter fw = new FileWriter(file.getAbsoluteFile());
+			BufferedWriter bw = new BufferedWriter(fw);
+			bw.write(content);
+			bw.close();
+ 
+			System.out.println("Written to the accounts file.");
+ 
+		} catch (IOException e) {
+			//e.printStackTrace();
+			frontend.error("Error: Unable to write to accounts file.");
+		}
+	}
+	public static void writeToSummaryFile(String content){
+		try {
+			 
+			File file = new File("./src/transaction_summary.txt");
+ 
+			// if file doesnt exists, then create it
+			if (!file.exists()) {
+				file.createNewFile();
+			}
+ 
+			FileWriter fw = new FileWriter(file.getAbsoluteFile());
+			BufferedWriter bw = new BufferedWriter(fw);
+			bw.write(content);
+			bw.close();
+ 
```

Comment:
possibly in a finally block, which should guarantee it to be closed even if the IO Exception is thrown.